src/serializers/shared.rs

```diff
@@ -345,3 +348,14 @@ pub(super) fn object_to_dict<'py>(value: &'py PyAny, is_model: bool, extra: &Ext
         Ok(attrs)
     }
 }
+
+pub(crate) fn slots_dc_dict(value: &PyAny) -> PyResult<&PyDict> {
+    let py = value.py();
+    let dc_fields: &PyDict = value.getattr(intern!(py, "__dataclass_fields__"))?.downcast()?;
```

Comment:
I'm pretty sure this will include init vars

Reply:
okay, I'll look.

Reply:
Confirmed, and it also includes ClassVars:
```python
from dataclasses import InitVar, dataclass
from typing import ClassVar


@dataclass
class A:
    x: InitVar[int]
    y: ClassVar[str]

print(A(x=1))
#> A()
print(A.__dataclass_fields__)
"""
{'x': Field(name='x',type=dataclasses.InitVar[int],default=<dataclasses._MISSING_TYPE object at 0x1054a2620>,default_factory=<dataclasses._MISSING_TYPE object at 0x1054a2620>,init=True,repr=True,hash=None,compare=True,metadata=mappingproxy({}),kw_only=False,_field_type=_FIELD_INITVAR), 'y': Field(name='y',type=typing.ClassVar[str],default=<dataclasses._MISSING_TYPE object at 0x1054a2620>,default_factory=<dataclasses._MISSING_TYPE object at 0x1054a2620>,init=True,repr=True,hash=None,compare=True,metadata=mappingproxy({}),kw_only=<dataclasses._MISSING_TYPE object at 0x1054a2620>,_field_type=_FIELD_CLASSVAR)}
"""
```

Reply:
i think this is fixed.